src/control/conf.c

```diff
@@ -439,7 +439,8 @@ gchar *dt_conf_read_values(const char *filename,
       const char* ret = fgets(line, LINE_SIZE, f);
       if(ret != NULL)
       {
-        g_strchomp(line);
+        // only remove trailing carriage return, newline but leave whitespace #14906
```

Comment:
Please no #ref in the code. The commit message is sufficient in this case. TIA.

Reply:
Done